sdk/tables/Azure.Data.Tables/tests/TableServiceClientLiveTests.cs

```diff
@@ -45,27 +45,103 @@ private TableServiceClient CreateTableServiceClient()
         /// Validates the functionality of the TableServiceClient.
         /// </summary>
         [Test]
-        public async Task GetTablesReturnsTables()
+        [TestCase(null)]
+        [TestCase(5)]
+        public async Task GetTablesReturnsTables(int? pageCount)
         {
-            string tableName = $"testtable{Recording.GenerateId()}";
+            string tableName = $"testtable{(IsAsync ? "async" : "sync")}{Recording.GenerateId()}";
```

Comment:
Helper method?

Reply:
But why do you need the suffix?

Reply:
> But why do you need the suffix?

I found that the `Recording.GenerateId()` method re-uses the same IDs across isAsync variation runs and so I ended up with name collisions during recording runs. I'm curious if there is a better approach for this.

Reply:
There is a feature in the generator that tries to reuse the last set of IDs to reduce the recording diff but it takes them from previous recording of this specific tests.

Can you try removing the old recordings before re recording and seeing if it helps?

Reply:
> Can you try removing the old recordings before re recording and seeing if it helps?

That worked, thanks!